man/systemd.socket.xml

```diff
@@ -771,6 +771,17 @@
         <xi:include href="version-info.xml" xpointer="v246"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>AllowFileDescriptorPassing=</varname></term>
```

Comment:
hmm, I think "allow" is the wrong verb. because that suggests this was a security thing, i.e. that the program couldn't turn this off trivially on its own. But it doesn't do that really. apps can set disable/enable this on their own.

maybe AcceptFileDescriptors=?

TakeFileDescriptors=?